styles/components/modal.css

```diff
@@ -0,0 +1,38 @@
+.modal-cont {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  padding-top: 8rem;
+  position: fixed;
+  z-index: 10;
+  background-color: #00000066;
```

Comment:
Use a variable here, as you are using in other places

Reply:
sure will do this